src/hotspot/share/adlc/forms.cpp

```diff
@@ -284,6 +285,7 @@ Form::DataType Form::is_store_to_memory(const char *opType) const {
   if( strcmp(opType,"StoreN")==0)  return Form::idealN;
   if( strcmp(opType,"StoreNKlass")==0)  return Form::idealNKlass;
   if( strcmp(opType,"StoreVector")==0 )  return Form::idealV;
+  if( strcmp(opType,"VectorMaskedStore")==0 )  return Form::idealV;
```

Comment:
Same comment but for store - what do you think about naming it StoreVectorMasked?